src/transformers/models/deberta/modeling_deberta.py

```diff
@@ -37,6 +38,8 @@
 
 logger = logging.get_logger(__name__)
 
+convert_to_dtype = not version.parse(torch.__version__) < version.parse("1.11")
```

Comment:
Maybe a better name for that value would be something more specific to its purpose, but then the name starts being long and @sgugger gets angry, wdyt?

```suggestion
convert_softmax_tensor_to_dtype = not version.parse(torch.__version__) < version.parse("1.11")
```

Reply:
I'd maybe just add a `do_convert...` to make it a bit clearer it's a flag and not a function

Reply:
don't mind the name after :-)